mesh/reward.go

```diff
@@ -8,12 +8,12 @@ import (
 
 type Config struct {
 	// reward config
-	BaseReward *big.Int
+	BaseReward *big.Int `mapstructure:"base-reward"`
 }
 
 func DefaultMeshConfig() Config {
 	return Config{
-		BaseReward: big.NewInt(5000),
+		BaseReward: big.NewInt(50*10 ^ 12),
```

Comment:
comment on how much it is actually in smeshes

Reply:
50